vite.config.ts

```diff
@@ -13,20 +13,35 @@ export default defineConfig( {
 	build: {
 		target: 'esnext',
 		lib: {
-			entry: 'src/index',
-			fileName: 'index',
-			formats: [ 'es', 'umd' ],
-			name: 'CKEDITOR_INTEGRATIONS_COMMON'
+			formats: [ 'es', 'cjs' ],
```

Comment:
This package needs a UMD build, as it's required by UMD builds of Vue and React integrations

Reply:
It's not possible to compile UMD multiple entry builds in Vite at this moment. I checked and it used to work normally.

Reply:
A `require` call will not work in a browser, so it's not a suitable substitute. We've made a decision (good or bad) to have UMD builds of Vue and React integrations, and thus we need UMD builds for dependencies too.

Vite exposes Rollup config that should allow for that.